docs/guides/lacp-install-upgrade.md

```diff
@@ -0,0 +1,140 @@
+# Using LACP to Net Install or Upgrade XCP-ng
+
+## Foreword
+
+:::warning
+
+- This guide is specifically for environments using LACP-only networks, without LACP fallback. It assumes you do not want to reconfigure your switches.
+- The recommended approach is to enable "LACP fallback" (or an equivalent feature) on your switch. This way you can simply use a single interface during installation or upgrade.
+- If you’re performing a new installation on an LACP-only network, be aware that your host will lose network connectivity after booting. You’ll need physical access or IPMI access to configure the network properly afterward. Xen Orchestra won’t be able to assist with bond creation at this stage.
+- For upgrades, your existing configuration will remain intact after the upgrade is done.
+
+:::
+
+Our installer does not support LACP. This guide explains how to bypass this limitation, but please read all the instructions carefully before proceeding, as there are important caveats.
+When using LACP, your switch needs to be configured for it. If you cannot or prefer not to reconfigure your switch, you won’t be able to use the single-interface option provided by the installer.
+For other bonding methods (active-backup, balance-lsb), the switch does not need to be aware of the bond, and you can simply use one of the interfaces that will be part of the bond.
+
+## Do you actually need network?
+
+A working network configuration is only necessary in the following scenarios:
+
+- New installation from net install ISO
+- Upgrading supporting hosts, even from a full ISO
```

Comment:
```suggestion
- Upgrading supporting hosts, even from a full ISO, as they need to contact the master host during the ugprade
```

Reply:
Done, I also added periods to these bullet point now that we have an actual sentence in there.